cluster/examples/kubernetes/ceph/upgrade-from-v1.1-apply.yaml

```diff
@@ -126,3 +126,92 @@ spec:
           properties:
             caps:
               type: object
+---
```

Comment:
I'm not sure if this file should be changed. @BlaineEXE ?

Reply:
do I need to create a new file for this one?

Reply:
I think @BlaineEXE will take care of it before we release 1.3. @BlaineEXE wdyt?

Reply:
Yes please create a file called `upgrade-from-v1.2-apply.yaml` so we can move forward with that patch.